website/docs/r/express_route_connection.html.markdown

```diff
@@ -95,6 +95,8 @@ The following arguments are supported:
 * `authorization_key` - (Optional) The authorization key to establish the Express Route Connection.
 
 * `enable_internet_security` - (Optional) Is Internet security enabled for this Express Route Connection?
+  
+* `enabled_express_route_gateway_bypass` - (Optional) Indicates whether FastPath enabled for Virtual Wan Firewall Hub. Defaults to `false`.
```

Comment:
Is that what this property does? The description doesn't sound like it matches the property name
```suggestion
* `enabled_express_route_gateway_bypass` - (Optional) Specified whether Fast Path is enabled for Virtual Wan Firewall Hub. Defaults to `false`.
```